app/grandchallenge/workstations/static/workstations/js/session-control.mjs

```diff
@@ -0,0 +1,134 @@
+'use strict';
```

Comment:
Is this necessary for modules?

Reply:
Ah apparently not (double checked [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#strict_mode_for_modules))